packages/darklang/stdlib/db.dark

```diff
@@ -0,0 +1,91 @@
+module Darklang =
+  module Stdlib =
+    module DB =
+      /// Upsert <param val> into <param table>, accessible by <param key>
+      let set (value: 'a) (key: String) (table: 'a) : 'a =
```

Comment:
Should this be `(table: DB<'a>)` instead of `(table: 'a)`? 
I don't think we have support for this syntax `DB<'a>`

Reply:
If tests are passing, then I think this is OK - can you just add CLEANUPs that they _should_ be `DB<'a>`-style? We can follow up on that when we get rid of LibParser